scooby/__init__.py

```diff
@@ -15,10 +15,11 @@
 ``watermark.py`` from https://github.com/rasbt/watermark.
 """
 
-from scooby.report import Report, get_version
-from scooby.knowledge import in_ipython, in_ipykernel
+from scooby.report import Inspection, Report, get_version
+from scooby.knowledge import get_standard_lib_modules, in_ipython, in_ipykernel
 
-__all__ = ['Report', 'in_ipython', 'in_ipykernel', 'get_version']
+__all__ = ['Inspection', 'Report', 'get_standard_lib_modules', 'in_ipython',
```

Comment:
I figured alphabetical sorting was best... why would order matter? Is there a situation where the order gives preference?

Reply:
I have a habit of sorting all imports by type (class, function, constant) then sorting them alphabetically - just something I have always done to make keeping track of things easier

Reply:
That is fine then. It can matter, but currently I think it doesn't for scooby. If you document with Sphinx, for instance, you can have Sphinx list functions in the order they appear in __all__. I Use this in my packages to have the most important functions/classes at the beginning of the API docs.

Reply:
Ah, I wasn't aware of that. Seems useful!